ingest/profiles/default/config.yaml

```diff
@@ -0,0 +1,3 @@
+cores: all
+printshellcmds: true
```

Comment:
_not asking for a change_ 

In case it's not clear, these configs are not necessary when running the builds with the Nextstrain CLI via `nextstrain build`. 

1. CLI will use all cores if the `--cpus` option is not provided ([ref](https://docs.nextstrain.org/projects/cli/en/stable/commands/build/#cmdoption-nextstrain-build-cpus)). 
2. The `--printshellcmds` flag is passed to snakemake by default ([ref](https://github.com/nextstrain/cli/blob/b8bbcaf37c17255ddb145d4aac6e4f3b25007d34/nextstrain/cli/command/build.py#L209))